homeassistant/components/config/config_entries.py

```diff
@@ -54,23 +54,29 @@ class ConfigManagerEntryIndexView(HomeAssistantView):
         """List available config entries."""
         hass = request.app["hass"]
 
-        return self.json(
-            [
+        results = []
+
+        for entry in hass.config_entries.async_entries():
+            handler = config_entries.HANDLERS.get(entry.domain)
+            supports_options = (
+                hasattr(handler, "async_get_options_flow")
```

Comment:
We know that all handlers have this method, right? We added it to the base config flow class.

Reply:
So that was what actually broke 😄 

Registering a discovery or webhook config entry will actually register it using a partial instead of extending the config flow class. And partial does not have `async_get_options_flow`, failing the command.

Reply:
Removing this line will now fail the test with this:

```
  File "/Users/paulus/dev/hass/home-assistant/homeassistant/components/config/config_entries.py", line 62, in get
    handler.async_get_options_flow
AttributeError: 'functools.partial' object has no attribute 'async_get_options_flow'
```

Reply:
New idea, reworking to remove the use of partial.